parseAPI/src/IA_power.C

```diff
@@ -309,91 +309,9 @@ bool IA_power::isReturnAddrSave(Address& retAddr) const
   return ret;
 }
 
-bool IA_power::isReturn(Dyninst::ParseAPI::Function * context, Dyninst::ParseAPI::Block* currBlk) const
+bool IA_power::isReturn(Dyninst::ParseAPI::Function *, Dyninst::ParseAPI::Block*) const
 {
-  /* Check for leaf node or lw - mflr - blr pattern */
```

Comment:
Is this pattern no longer being used in powerpc so we don't need to check it anymore?

Reply:
Correct. This is from ancient POWER and hasn't been used since v2.0 (~Power7) which is the oldest we support.